packages/server/utils/analytics/analytics.ts

```diff
@@ -107,8 +108,22 @@ class Analytics {
     })
   }
 
-  meetingStarted = (userId: string, meeting: Meeting, template?: MeetingTemplate) => {
-    this.track(userId, 'Meeting Started', createMeetingProperties(meeting, undefined, template))
+  meetingStarted = (
+    userId: string,
+    meeting: Meeting,
+    template?: MeetingTemplate,
+    isRecurring?: boolean
+  ) => {
+    this.track(userId, 'Meeting Started', {
+      isRecurring: isRecurring,
+      ...createMeetingProperties(meeting, undefined, template)
+    })
+  }
+
+  recurrenceStarted = (userId: string, meeting: Meeting) => {
+    // :TODO: (jmtaber129): Add properties related to recurrence settings (duration, frequency,
+    // etc.) after we support configuring those.
+    this.track(userId, 'Meeting Recurrence Started', createMeetingProperties(meeting))
```

Comment:
To my point above (if you agree with me), here instead of the properties for the first meeting, we should log properties for the Meeting Series table. As v1, let's have:
1. `meetingSeriesId` so we can join against meeting events
2. `teamId`
3. `meetingType`

Reply:
So since this is effectively just a normal meeting event (like meeting started, meeting joined etc.), we'll get `teamId` and `meetingType` as part of `createMeetingProperties`, and I've also just added `meetingSeriesId` to that helper as well.